.flowconfig

```diff
@@ -9,6 +9,7 @@
 [libs]
 
 [options]
+module.name_mapper.extension='less' -> '<PROJECT_ROOT>/NullModule.js'
 module.name_mapper='^console\/\(.*\)$' -> '<PROJECT_ROOT>/console/\1'
-module.name_mapper='.*\(.less\)$' -> 'empty/object'
```

Comment:
I believe this would have worked if we had a dependency on https://www.npmjs.com/package/empty

Reply:
Yeah, I tried that after some searching, but it didn't work. Flow has been a real pain around "things that should work but don't, and no warnings or errors are thrown."